### DISCUSSION TITLE
Is it possible for OGB to support CogDL?

### DISCUSSION THREAD

**Discussion by Diego0511** [snap-stanford/ogb#326] [General]
Hi,

I am a member of the CogDL development team.(CogDL is an Extensive Toolkit for Deep Learning on Graphs developed by Tsinghua University) And I am wondering if it would be possible for OGB to support CogDL to quickly get an OGB dataset object like PYG and DGL.

Thank you!

**Comment by weihua916** [comment]
Hi! Thanks for reaching out and for sharing your exciting new graph learning package.
For now, we only support the two most popular packages, PyG and DGL (both ~10K stars), but we recognize there are many emerging packages (such as yours!). For them, we recommend using our library-agnostic dataset object.
